### ISSUE TITLE
Placing a firewall pipe causes server crash and savegame corruption

### ISSUE THREAD

**Issue by Tyniann** [RS485/LogisticsPipes#726]
I tried setting up a block-compression system using lp+cyclic assemblers. I've done it before, so I know how to do it, but when I placed down the firewall pipe to block the crafting requests the server immediately shut down and kept crashing on restart. I had to delete the firewall pipe via mcedit, then the save worked again.

After that I did some extra testing in a new singleplayer world and I discovered that the system works fine. No crashes, no bugs. As soon as I moved this save on the server though things started to get weird again, meaning the firewall pipe didn't prevent the crafting requests from outside. Blocks from the chest got autorequested and autocrafted into ingots, just to get compressed again and so on. It totally ignored that the firewall pipe was even there. I tried blocking power flow just to see if it affected the connected pipes and that worked fine, so it seems it's just the crafting requests that somehow manage to sneak by the firewall pipe. Well, at least it didn't crash and corrupted the savegame this time.

I'm using the FTB infinity 1.7 modpack without any personal changes and of course minecraft 1.7.10.
A link to the crashlog can be found here: https://gist.github.com/theZorro266/7798ff2e1259ca130acc (original: https://www.dropbox.com/s/j046kyhfr6ybp9e/crash-2015-06-30_12.44.16-server.txt?dl=0)

Should any additional informations be required just say so. I'd be happy to help.

**Comment by hron84** [comment]
Could you please post also a screenshot about your setup? Just to show us how it looks like ingame.

**Comment by Tyniann** [comment]
Sure.

I made a buffer chest with 3 supplier pipes connected to the rest of the lp system requesting the ingots. Once they are in the buffer chest Ender IO takes over and delivers the ingots to the cyclic assemblers, which eject the blocks into some basic buildcraft pipes and transports them to the block-storage chests. These chests are connected via provider pipes to the main system. I marked the place where I put the firewall pipe with a golden transport pipe.
![2015-06-30_16 28 04](https://cloud.githubusercontent.com/assets/13117389/8433372/a07a3172-1f45-11e5-9023-0bdb8709f98a.png)
By the way: I also did try to place the firewall pipe right after the provider pipes on the block-storage chests. If I do that, the server won't crash but even if I block crafting requests they happily send their contents through the firewall pipe into the next autocrafting table. Of course it's possible that I overlooked something, but since I did use this setup in the past I'm pretty confident I did it right.

The configuration of the supplier lp, that requests all the stuff I want to craft into blocks:
![2015-06-30_16 28 58](https://cloud.githubusercontent.com/assets/13117389/8433371/a07be454-1f45-11e5-866d-470480bab903.png)
